tests/test_app.py

```diff
@@ -0,0 +1,53 @@
+from unittest import mock
+
+import importlib_resources
+import jwt
+import pytest
+
+from gidgethub import (
+    app as gh_app,
```

Comment:
For some reason `app` is clashing with pytest-tornasync fixture, so I had to rename this to `gh_app` in tests.
I could also call the module `apps` instead of `app` if that's better? (just like `actions` with `s`)

Reply:
Naming it 'apps' is fine by me.